### DISCUSSION TITLE
旧版本的数据库文件如何迁移到目前的docker版本

### DISCUSSION THREAD

**Discussion by suppergit** [usememos/memos#183] [Q&A]
使用旧版本的数据库文件memos_release.db重命名为memos_prod.db后更新docker，登录不了(好像新版本的要邮箱登录，旧版本是用户名登录)，请问要如何迁移数据到新版本的docker。

**Comment by boojack** [comment]
如果旧版本是 `0.0.x` 或者 `0.1.x`，可能无法直接升级，可以先加一下 tg 用户群。

**Reply by lcjssg** [reply]
额，请问tg群号

**Reply by boojack** [reply]
Link: <a href="https://t.me/+-_tNF1k70UU4ZTc9">Discuss in Telegram 👾</a>
